Module not found: Error: Cannot resolve module 'oauth'

I'm building a project which is a fork of [electron-react-boilerplate](https://github.com/chentsulin/electron-react-boilerplate).

I have added the `github4` module to the `packages.json` file and ran `npm i` but when I start the server with `npm run hot-server` I have the following error :

```
ERROR in ./~/github4/test/oauth.js
Module not found: Error: Cannot resolve module 'oauth' in /home/zedtux/Developments/grrn/node_modules/github4/test
 @ ./~/github4/test/oauth.js 19:13-29
```

Of course, adding the oauth module to the `package.json` file solve the issue .. but it should be included as a dependency, shouldn't be ?

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "github4",
-  "version": "0.2.8",
+  "version": "0.2.9",
   "description": "NodeJS wrapper for the GitHub API",
   "author": "Mike de Boer <info@mikedeboer.nl>",
   "contributors": [
@@ -28,12 +28,12 @@
   "dependencies": {
     "follow-redirects": "0.0.7",
     "https-proxy-agent": "^1.0.0",
-    "mime": "^1.2.11"
+    "mime": "^1.2.11",
+    "oauth": "^0.9.14"
   },
   "devDependencies": {
     "async": "^0.9.0",
     "mocha": "~1.13.0",
-    "oauth": "~0.9.7",
     "optimist": "~0.6.0",
     "should": "^8.0.2"
   },

